### ISSUE TITLE
[bug]: Blank Browser When Launching

### ISSUE THREAD

**Issue by Blootrix** [invoke-ai/InvokeAI#2259]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Windows

### GPU

cuda

### VRAM

12GB

### What happened?

Launch invoke.bat > Option 2 > Open http://127.0.0.1:9090/ > Blank screen.

It was working earlier toady and has now stopped working. When it stopped working with Chrome, I tried launching it with Edge which worked once and hasn't worked since.

Tried running update.bat
Tried running as Admin
Restarting does not fix anything

Last 3 lines of code
>> Point your browser at http://127.0.0.1:9090
>> System config requested
>> Patchmatch initialized

**Comment by Lollipopkiller** [comment]
I literally have the same problem you found any fix yet?

**Comment by Blootrix** [comment]
> I literally have the same problem you found any fix yet?

No, I've reinstalled it twice and the issue comes back, even with a completely fresh install and even when trying different browsers. I have no idea why.

If you find a fix, please let me know.

**Comment by Lollipopkiller** [comment]
sure

**Comment by Blootrix** [comment]
> sure

Out of curiosity, do you use either an LG TV as a monitor or use scaling on your system?
I think it might be related to that, but I'm not sure.

**Comment by tyk-adrian** [comment]
I have the same problem–need to always refresh the page to get it to work. Initial load will always be blank with this error when I open the dev console:
```
index.be0f03f1.js:32 TypeError: Cannot read properties of undefined (reading 'description')
    at lt.memoizeOptions.resultEqualityCheck (index.be0f03f1.js:479:52310)
    at index.be0f03f1.js:410:83300
    at d (index.be0f03f1.js:410:82251)
    at index.be0f03f1.js:410:83432
    at d (index.be0f03f1.js:410:82251)
    at l (index.be0f03f1.js:471:453)
    at index.be0f03f1.js:471:660
    at Object.useSyncExternalStore (index.be0f03f1.js:30:26615)
    at rn.useSyncExternalStore (index.be0f03f1.js:9:6483)
    at TG.useSyncExternalStoreWithSelector (index.be0f03f1.js:471:729)
```

**Comment by hossainiir** [comment]
I have same issue! Fresh installing didn't work.

**Comment by hossainiir** [comment]
I found the problem by running the frontend as dev!
https://github.com/invoke-ai/InvokeAI/blob/5f16148dea45cf42f4e637d802dd1261a436f8b0/frontend/src/features/system/components/ModelSelect.tsx#L26

I don't know why but seems `model_list` is empty!

**Comment by sashatc** [comment]
Happened to me when I chose stable-diffusion-1.5 to be used
then this appears in console while starting up
** "stable-diffusion-1.5" is not a known model name; falling back to stable-diffusion-1.5.

**Comment by jibberishballr** [comment]
@sashatc I'm confused. Did you take the models.example and models.yaml file and from the config file and put them in the InvokeAI root directory or am I missing something?

@hossainiir how did you fix the model list being empty?

**Comment by sashatc** [comment]
@jibberishballr I am not sure if you have the same cause. In my case it was stable-diffusion-1.5 being configured (still do not know where the current model selection is stored) as selected model, but it somehow disappeared as a model from configs/models.yaml, so I had to edit configs/models.yaml file to add it back there manually.

**Comment by jibberishballr** [comment]
> @jibberishballr I am not sure if you have the same cause. In my case it was stable-diffusion-1.5 being configured (still do not know where the current model selection is stored) as selected model, but it somehow disappeared as a model from configs/models.yaml, so I had to edit configs/models.yaml file to add it back there manually.

Ah I follow you. Yes, don't have that issue of the config file missing from the yaml file. 

I just have a blank browser that never loads the Invoke AI UI.

**Comment by sashatc** [comment]
@jibberishballr - blank browser is the resulting behavior. If you look at browser's developer console - what do you see? Do you see `index.be0f03f1.js:32 TypeError: Cannot read properties of undefined (reading 'description')
    at lt.memoizeOptions.resultEqualityCheck (index.be0f03f1.js:479:52310)
    at index.be0f03f1.js:410:83300
    at d (index.be0f03f1.js:410:82251)
    at index.be0f03f1.js:410:83432
    at d (index.be0f03f1.js:410:82251)
    at l (index.be0f03f1.js:471:453)
    at index.be0f03f1.js:471:660
    at Object.useSyncExternalStore (index.be0f03f1.js:30:26615)
    at rn.useSyncExternalStore (index.be0f03f1.js:9:6483)
    at TG.useSyncExternalStoreWithSelector (index.be0f03f1.js:471:729)`
error as mentioned by @tyk-adrian above?

Do you also see repeating messages in your InvokeAI cmd console like this every time you try to reload browser
`>> System config requested
>> Patchmatch initialized
>> System config requested
>> System config requested
>> System config requested
>> System config requested
>> System config requested
>> System config requested
>> System config requested
>> System config requested`

???

**Comment by naralysis** [comment]
have you guys try copy past the localhost address to other browser like firefox?

**Comment by BayerHeroine** [comment]
I fix this, just connect to 127.0.0.1:9090 instead localhost:9090
